tests/resources/testrepo/.gitted/worktrees/testrepo-worktree/gitdir

```diff
@@ -0,0 +1 @@
+/home/pks/Development/elego/git/libgit2-worktree/tests/resources/testrepo-worktree/.git
```

Comment:
Can this be a relative path?  If not, we'll have to figure out some mechanism here where we can rewrite this to the sandbox path.  We do something similar with the submodule tests (see [submodule_helpers.c](https://github.com/libgit2/libgit2/blob/master/tests/submodule/submodule_helpers.c)).